packages/core/core-flows/src/common/steps/create-entities.ts

```diff
@@ -0,0 +1,50 @@
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+export interface CreateEntitiesStepType {
+  moduleRegistrationName: string
+  invokeMethod: string
+  compensateMethod: string
+  entityIdentifier?: string
+  data: any[]
+}
+
+export const createEntitiesStepId = "create-entities-step"
+/**
+ * This step creates entities for any given module or resource
+ */
+export const createEntitiesStep = createStep(
```

Comment:
I think it is ok to have this generic step, but I believe the ultimate solution will be modules auto generating their methods as steps.
That would cleanup a lot of steps we have and custom modules would also have their methods as steps by default.